ios/RCTOneSignal/RCTOneSignal.m

```diff
@@ -60,6 +60,11 @@ - (id)initWithLaunchOptions:(NSDictionary *)launchOptions appId:(NSString *)appI
     return self;
 }
 
+// This isn't required, the iOS native SDK already hooks into this event.
++ (void)didReceiveRemoteNotification:(NSDictionary *)dictionary {
```

Comment:
Why is this happening, if this isn't required and the iOS native SDK already hooks into this event?